providers/bnxt_re/main.c

```diff
@@ -110,13 +110,17 @@ static const struct verbs_context_ops bnxt_re_cntx_ops = {
 
 /* Context Init functions */
 static struct verbs_context *bnxt_re_alloc_context(struct ibv_device *vdev,
-						   int cmd_fd)
+						   int cmd_fd,
+						   void *vendor_attr)
 {
 	struct ibv_get_context cmd;
 	struct ubnxt_re_cntx_resp resp;
 	struct bnxt_re_dev *dev = to_bnxt_re_dev(vdev);
 	struct bnxt_re_context *cntx;
 
+	if (vendor_attr)
```

Comment:
Why? If 'vendor_attr' is internal to the driver we don't need to check it everywhere like this.